app/assets/stylesheets/components.scss

```diff
@@ -115,15 +115,52 @@
 }
 
 .circuit-card-name {
-  display: inline-block;
   float: left;
   font-weight: 600;
-  margin-top: 10px;
-  max-height: 30px;
-  max-width: 150px;
-  overflow: hidden;
+  margin-top: 5px;
+  position: relative;
   text-align: left;
-  text-overflow: ellipsis;
+
+  p {
+    max-height: 50px;
+    max-width: 150px;
+    overflow: hidden;
+  }
+
+  .tooltiptext {
+    background-color: var(--green);
```

Comment:
Use sass variables defined in theme.scss
Also decrease the font size of tooltip text

Reply:
made changes accordingly!!